code/FulpstationCode/fulp_mobs/eclatmob.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/simple_animal/pet/cat/eclat //for a good pal, Fingus

```

Comment:
this should be a subtype of a cak

Reply:
very much this - its almost the same thing as the cak